install/policies/mdm.php

```diff
@@ -43,7 +43,7 @@
       'type_data'                           => '',
       'unicity'                             => 1,
       'plugin_flyvemdm_policycategories_id' => $category,
-      'comment'                             => __('Use TLS.', 'flyvemdm'),
+      'comment'                             => __('Use TLS', 'flyvemdm').'.',
```

Comment:
put the dot in the localizable string

Reply:
Why? the dot it's at the end and we could use the same translation of the upper line or do you really want to have to translate two string just for only one dot?

Reply:
Maybe in some languages the punctuation may be very different to latin languages. We cannot do assumption about locales.

Moreover this policy willl change its name see #494 We wil need to improve comment and probably use longer text than only copying the name.